### ISSUE TITLE
Enhancement Request - Support for Redirect Rules

### ISSUE THREAD

**Issue by acds** [multiplegeorges/vue-cli-plugin-s3-deploy#95]
Would it be possible to pick up all the routes in the Vue Router and generate a set of redirect rules similar to:

```XML
<?xml version="1.0"?>
<RoutingRules>
  <RoutingRule>
    <Condition>
      <KeyPrefixEquals>home</KeyPrefixEquals>
    </Condition>
    <Redirect>
      <ReplaceKeyWith>index.html</ReplaceKeyWith>
    </Redirect>
  </RoutingRule>
</RoutingRules>
```

for each route defined, to that browser refreshes work better given SPA/PWA setups, and pass that into the S3 bucket Website hosting configuration on create an update in on deploy ?

**Comment by nicekiwi** [comment]
If S3 supports this I don't see why not.

**Comment by acds** [comment]
There is a REST api for doing this...
https://docs.aws.amazon.com/AmazonS3/latest/dev/how-to-page-redirect.html

**Comment by nicekiwi** [comment]
Actually this has already been implemented in @next. via the `staticWebsiteConfiguration` option.

**Comment by nicekiwi** [comment]
I'll add an option to update the static config.

**Comment by acds** [comment]
@nicekiwi thanks, not sure what @next is, and any updated tot he documentation on how to use `staticWebsiteConfiguration` option and how that is configured with this package would be great.

What's the best eta for the 4.1.0 release - no Due date currently listed.

**Comment by nicekiwi** [comment]
It will be available in 4.0.0 or 4.0.0-rc3. I can't give a date, soon.
